### ISSUE TITLE
Share it as a package

### ISSUE THREAD

**Issue by lvignoli** [lvignoli/diapo#3]
See typst/pacakges.

**Comment by saadulkh** [comment]
Still interested in this? We can look into it.

**Comment by lvignoli** [comment]
Hi @saadulkh! I thought about it but I was in the mindset that it was too small for being a proper package, and could be just a template.
But there is no warm in doing it, especially if there are regular users such as you :) Let's do it!

**Comment by saadulkh** [comment]
You are right. I was going through the submission [guidelines](https://github.com/typst/packages#submission-guidelines) and currently, publishing templates as packages is discouraged. See this note:

> Note: Please do not submit templates as packages just yet. We plan to build infrastructure around this so that they can show up in the web app's template gallery and be used to scaffold a project through the CLI. Stay tuned!

So, I think it's better to wait till the proper implementation. Till then, lets just use this template as a [local](https://github.com/typst/packages#local-packages) package

Currently, we have only one function i.e `transition` that can classify this as a package. But it can be a hit or miss. Tho, we can give it a try if you want.

Thanks for the quick response tho :)

**Comment by saadulkh** [comment]
I have structured it as a [package](https://github.com/saadulkh/diapo/tree/package).
Also, it's not clear what is called a template and whats' a proper package. Because, [another](https://github.com/typst/packages/tree/main/packages/preview/polylux/0.2.0) related library is also available in packages.
So, we can PR this as well when you are ready.

**Comment by lvignoli** [comment]
polylux is clearly much more powerful and feature-rich (there is an API for slide "animation") so it does qualify as a package to me.
diapo seems a bit too thin to be a proper package indeed, but local packages are nice! Please open a PR so I can merge your branch :)

**Comment by lvignoli** [comment]
I'll change the release process to a zip file with the proper file structure to install locally
